### DISCUSSION TITLE
List or invalidate all passwords a given user has access to

### DISCUSSION THREAD

**Discussion by rschueler** [dani-garcia/vaultwarden#3202] [Ideas]
It would be highly beneficial if there were a procedure or a report in place to address internal password breaches. In the event that a company member is departing, and all of the passwords they had access to need to be changed, a report would be extremely useful. Even more optimal would be a front-end process that can filter all revoked passwords.

**Comment by BlackDex** [comment]
I suggest to bring this feature up at Bitwarden. This can only be checked on the clients.
